Source/Diagnostics/BTDiagnostics.cpp

```diff
@@ -98,6 +108,14 @@ void BTDiagnostics::DerivedInitData ()
         DefineCellCenteredMultiFab(lev);
     }
 
+    /** Allocate vector of particle buffer vectors for each snapshot */
```

Comment:
Since this is a body, we cannot add declarative doxygen here:
```suggestion
    /* Allocate vector of particle buffer vectors for each snapshot */
```